packages/tailwindcss/src/utilities.test.ts

```diff
@@ -16385,28 +16386,28 @@ test('@container', async () => {
       '@container-[size]/sidebar',
     ]),
   ).toMatchInlineSnapshot(`
-    ".\\@container {
-      container-type: inline-size;
+    ".\\@container-\\[size\\]\\/sidebar {
+      container: sidebar / size;
     }
 
-    .\\@container-\\[size\\] {
-      container-type: size;
+    .\\@container-normal\\/sidebar {
+      container: sidebar;
     }
 
-    .\\@container-\\[size\\]\\/sidebar {
-      container: sidebar / size;
+    .\\@container\\/sidebar {
+      container: sidebar / inline-size;
     }
 
-    .\\@container-normal {
-      container-type: normal;
+    .\\@container {
+      container-type: inline-size;
     }
 
-    .\\@container-normal\\/sidebar {
-      container: sidebar;
+    .\\@container-\\[size\\] {
+      container-type: size;
     }
 
-    .\\@container\\/sidebar {
-      container: sidebar / inline-size;
+    .\\@container-normal {
+      container-type: normal;
```

Comment:
Ah nice so this previously counted `container` and `container-type` as the same but isn't doing that anymore. Neat!